wiki/Help_Centre/en.md

```diff
@@ -55,29 +51,55 @@ Good information to include this email would be:
 
 If this succeeds, you will be able to reset your own password without the assistance of the support staff.
 
-If this does not work for you, please send us a support ticket immediately to [accounts@ppy.sh](mailto:accounts@ppy.sh) with as much identifying information as possible that will help us establish you as the owner of your account.
+If this does not work for you, please send us a support ticket immediately to [accounts@ppy.sh](mailto:accounts@ppy.sh) with as much identifying information as possible that will help us identify you as the owner of your account.
 
 Emailing from your registration address is a huge bonus here, as well as any other information you can provide that only the holder of the account would know.
 
 Please do not include replay or score data in your ticket - we cannot use this to verify your ownership of the account.
 
+#### What is this 'Bancho authentication error' I keep on receiving?
+
+**This typically happens when there are networking issues between you and our servers.**
+
+There's a few things we can try to resolve this issue:
+
+##### Have you entered your password correctly?
+
+**This is the most obvious one.**
+
+If you've forgotten what your password is, you can use the [password recovery page](https://osu.ppy.sh/home/password-reset) to recover access to your account.
+
+##### Are you using any proxy servers or VPNs?
+
+**Disabling these before attempting to connect again will likely fix your issue.**
+
+Some VPN providers and proxy servers simply do not work with osu! for whatever reason.
+
+##### I'm still getting the error!
+
+**We'll need a `network.log` file from you to figure out what's going on if neither of the above solutions work.**
+
+Check out the article at the bottom of this page on how to get a log file to send to us.
```

Comment:
a direct link right here would really help

Reply:
I agree. Would this: [log files](/wiki/Help_Centre#log_files) be the correct format? just double checking